session.go

```diff
@@ -449,6 +449,13 @@ func (s *session) WriteBytesArray(pkgs ...[]byte) error {
 		return s.WriteBytes(pkgs[0])
 	}
 
+	// reduce syscall and memcopy for multiple packages
+	if _, ok := s.Connection.(*gettyTCPConn); ok {
+		if _, err := s.Connection.send(pkgs); err != nil {
+			return perrors.Wrapf(err, "s.Connection.Write(pkgs num:%d)", len(pkgs))
+		}
+	}
+
```

Comment:
`WriteBytesArray()` is only supported by tcp connection,  the code below is not needed.

Reply:
> `WriteBytesArray()` is only supported by tcp connection, the code below is not needed.

Looks like websocket also used it.